examples/helm-on-kubernetes-example/readme.md

```diff
@@ -38,16 +38,36 @@ Finally, let's use Helm to deploy it into the cluster:
 helm install helm-example ./target/classes/META-INF/dekorate/helm/<chart name>
 ```
 
+The above command will use the default values file `values.yaml`, we can select a different values file by doing:
```

Comment:
This is not clear. I suggest to say "To override the default values, part of the `values.yaml` file, pass as parameter you own value file `-f /path/to/the/file/values.dev.yaml` or set them using `--set key=val --set key=val ...`"

Reply:
Changed